Detections/ASimNetworkSession/IPEntity_imNetworkSession.yaml

```diff
@@ -0,0 +1,62 @@
+id: e2399891-383c-4caf-ae67-68a008b9f89e
+name: TI map IP entity to _Im_NetworkSession
+description: |
+  'Identifies a match in _Im_NetworkSession from any IP IOC from TI'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: ThreatIntelligenceTaxii
+    dataTypes:
+      - ThreatIntelligenceIndicator
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+query: |
+  let dt_lookBack = 1h;
+  let ioc_lookBack = 14d;
+  let IP_TI=ThreatIntelligenceIndicator
+  | where TimeGenerated >= ago(ioc_lookBack) and ExpirationDateTime > now()
+  | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId
+  | where Active == true
+  // As there is potentially more than 1 indicator type for matching IP, taking NetworkIP first, then others if that is empty.
+  // Taking the first non-empty value based on potential IOC match availability
+  | extend TI_ipEntity = coalesce(NetworkIP, NetworkDestinationIP, NetworkSourceIP,EmailSourceIpAddress,"NO_IP")
+  // Picking up only IOC's that contain the entities we want
+  | where TI_ipEntity != "NO_IP";
+  let IP_TI_list=toscalar(IP_TI | summarize make_set( TI_ipEntity));
+  IP_TI
+  // using innerunique to keep perf fast and result set low, we only need one match to indicate potential malicious activity that needs to be investigated
+  | join kind=innerunique (
```

Comment:
Sorry. Missed an important comment here: you need to lookup either src or dst ip. It does complicate the query.